src/core/multifeaturelistmodelbase.cpp

```diff
@@ -703,4 +737,6 @@ void MultiFeatureListModelBase::geometryChanged( QgsFeatureId fid, const QgsGeom
       break;
     }
   }
+
+  emit selectedCountChanged();
```

Comment:
Are we abusing this signal here?

Reply:
@m-kuhn , a bit yeah but I think it's OK here.